### PR TITLE
Geanyvc fix

### PR DESCRIPTION
**Pull request by kugel-**
please merge the fixes for 1.26

### CODE REVIEW

File: geanyvc/src/geanyvc.c [hunk 1]
```diff
@@ -1374,7 +1367,7 @@ create_commitDialog(void)
 	gtk_window_set_type_hint(GTK_WINDOW(commitDialog), GDK_WINDOW_TYPE_HINT_DIALOG);
 	gtk_dialog_set_has_separator(GTK_DIALOG(commitDialog), FALSE);
 
-	dialog_vbox1 = GTK_DIALOG(commitDialog)->vbox;
+	dialog_vbox1 = gtk_dialog_get_content_area (GTK_DIALOG(commitDialog));
```
Comment (b4n) [inline geanyvc/src/geanyvc.c]: style: no space before the opening `(`

File: geanyvc/src/geanyvc.c [hunk 2]
```diff
@@ -1464,7 +1457,7 @@ create_commitDialog(void)
 	gtk_box_pack_end(GTK_BOX(commit_text_vbox), lineColumnLabel, TRUE, TRUE, 0);
 	gtk_widget_show(lineColumnLabel);
 
-	dialog_action_area1 = GTK_DIALOG(commitDialog)->action_area;
+	dialog_action_area1 = gtk_dialog_get_action_area (GTK_DIALOG(commitDialog));
```
Comment (b4n) [inline geanyvc/src/geanyvc.c]: ditto

**Comment by b4n** [comment]
not tested yet, but LGTM if it works

**Comment by frlan** [comment]
Can somebody with Gtk3 please run a test? With Gtk2 it looks fine for me

**Comment by kugel-** [comment]
I found that the commit dialog hangs on GTK3. I'm checking this. However, this is not a regression so, technically it's still better as the status-quo as diff actions work.

**Comment by kugel-** [comment]
I double checked gtk3 again and fixed up the commit dialog. gtk2 build looks fine as well.

One more note regarding bf20fd7: I want this in because the old code was inherently incompatible with my splitwindow2 work, and it was strange/awkward and fragile regardless. Since I would like splitwindow2 to not break any of G-P this fix must be applied.

File: geanyvc/src/geanyvc.c [hunk 3]
```diff
@@ -392,23 +390,18 @@ show_output(const gchar * std_output, const gchar * name,
 			doc = document_new_file(name, ftype, std_output);
 			/* To due the given line is Scintilla's line number, but
 			 * we need the view line number in this case.  */
```
Comment (frlan) [inline geanyvc/src/geanyvc.c]: you should purge the comment too

**Pull request merged**